typings-checker/untyped-stylable-feature.tsx

```diff
@@ -2,12 +2,13 @@ import * as React from "react";
 import {stylable} from "../dist/src/index";
 
 interface CompProps {
+    message: string;
 }
 
 // assume stylesheet is not typed
 declare const style: any;
 
-// $ExpectType (props: { children?: ReactNode; }) => Element
+// $ExpectType (props: any) => Element
```

Comment:
@amir-arad  So it looses the `{message: string}` and gets `any` instead?

Reply:
it never had `{message: string}` in this scenario 
see the entire file for the different ways to use the API type-wise